Lib/test/test_support.py

```diff
@@ -161,6 +163,23 @@ def test_temp_dir__existing_dir__quiet_true(self):
                                         f'temporary directory {path!r}: '),
                         warn)
 
+    @unittest.skipUnless(hasattr(os, "fork"), "test requires os.fork")
+    def test_temp_dir__forked_child(self):
+        """Test that a forked child process does not remove the directory."""
```

Comment:
No other doc string in this file contains a reference to bpo. Therefore I added the reference as a comment.